apps/minifront/src/components/ibc/ibc-in/wallet-connect-button.tsx

```diff
@@ -52,19 +52,27 @@ interface BaseProps {
 
 const WalletButtonBase = ({ buttonText, isLoading, isDisabled, onClick }: BaseProps) => {
   return (
-    <Button
-      fluidWidth
-      size='md'
-      isLoading={isLoading}
+    <button
```

Comment:
UI is nearly the same, however removed the dependency on using the `interchain-ui` component.

Same as before:

<img width="393" alt="Screenshot 2024-06-12 at 4 08 45 PM" src="https://github.com/penumbra-zone/web/assets/16624263/a40523ba-e417-4c31-b73a-a0eada23bc2f">

New disabled color:

<img width="364" alt="Screenshot 2024-06-12 at 4 10 08 PM" src="https://github.com/penumbra-zone/web/assets/16624263/ffee3c04-81cb-4e19-9abd-4832681f4ace">

And added hover cursors depending on state.